lux/core/frame.py

```diff
@@ -695,7 +699,7 @@ def rec_to_JSON(recs):
 				# delete DataObjectCollection since not JSON serializable
 				del rec_lst[idx]["collection"]
 		return rec_lst
-
```

Comment:
Remove unnecessary deletion and addition of whitespace and newlines throughout the PR.